### PR TITLE
Release 0.9.0

### PR DESCRIPTION
**Pull request by ibuenros**
Includes changelog for Gobblin 0.9.0

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,68 @@
+GOBBLIN 0.9.0
+-------------
+
+### Created Date: 12/13/2016
+
+## Highlights
```
Comment (ydai1124) [inline CHANGELOG.md]: Shall we highlight the refactoring change?

Comment (ibuenros) [inline CHANGELOG.md]: It seems mostly irrelevant for users (only developers care). But if you think it's important, I'll include it.

**Comment by lbendig** [comment]
Hi,
The docs for the OracleExtractor and TeradataWriter is missing ATM. Is it fine if I provide them till the end of this week?

**Comment by ibuenros** [comment]
Hi Lorand,
Yes, if you provide the docs by EOW we'll include them in the release notes.
Thanks!
Issac

**Comment by wosiu** [comment]
Hi,
Do I get it right, that there is no support for Kafka 0.9.0->HDFS ingestion with this release? 
Thanks!

**Comment by shirshanka** [comment]
@wosiu : That's correct, although I think we'll see this soon in trunk. You should be able to consume from a 0.9 cluster using an 0.8 consumer though.

File: CHANGELOG.md (hunk 1 above)
Comment (shirshanka) [inline CHANGELOG.md]: I think the main change is for other projects that have a dependency on gobblin artifacts. So might be worth highlighting the change.

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,68 @@
...
+* Retention job is more generic, allowing arbitrary actions on dataset versions (https://gobblin.readthedocs.io/en/latest/data-management/Gobblin-Retention).
+* Docker integration (https://gobblin.readthedocs.io/en/latest/user-guide/Docker-Integration).
+* Gobblin jobs can be run embedded into other applications (http://gobblin.readthedocs.io/en/latest/user-guide/Gobblin-as-a-Library/).
+* Gobblin jobs can be run from CLI with full support for templates, plugins, etc. (http://gobblin.readthedocs.io/en/latest/user-guide/Gobblin-CLI/)
+* Topology based data replication: users can specify a topology for their data copy in config store, Gobblin Distcp will handle replication. (PR 1278, PR 1306, PR 1328, PR 1405)
+* Prioritization of work units when there is more work than can be run in a single job (PR 1283).
+* Enabled speculative excecution in MR mode (PR 1347).
+
+## NEW FEATURES
+
```
Comment (chavdar) [inline CHANGELOG.md]: Can you add:

[Sources] [PR 1422]  Google webmaster source

Comment (chavdar) [inline CHANGELOG.md]: Also @enjoyear in the acknoledgements .

**Pull request merged**